pkg/rtc/types/protocol_version.go

```diff
@@ -16,7 +16,7 @@ package types
 
 type ProtocolVersion int
 
-const CurrentProtocol = 12
+const CurrentProtocol = 13
```

Comment:
i don't think the clients with 12 has been released yet. this can probably still be added to that version unless v2 client won't support this.

Reply:
Good point. Will change it back.

Reply:
Actually, `client-sdk-js` has that version. There is no release, but internal use/e2e tests checkout latest client code. 

@lukasIO Can the `LeaveRequest` changes bundled within protocol 12 or should we bump it? Bumping makes it easier to deal with this. Otherwise, we have to coordinate client changes with merging this.

Reply:
would have to double check with the rest of the client team, but if there's no significant drawback, I'd opt to bump the protocol version to decouple these changes from the next release

Reply:
Can't think of any drawback (@paulwe WDYT?). Just a quick bump since the previous one.

Reply:
one more thought, protocol v12 is tied to a major release of the client sdks as `sid` will become async. 
So backporting that change to the `v1.x` versions won't be possible. But can't think of a way how we could make that work, so guess it will only be available `>=v2.x`

Reply:
i assumed it was all going to be added to the same release but whatever you all think is best

Reply:
Sounds good. Thank you Paul.

Lukas, to keep this simple, will make protocol 13. Yes, this will be available only v2.0 clients. I think that is fine, right?

Reply:
yeah, totally fine!